siliconcompiler/tools/vpr/vpr.py

```diff
@@ -26,6 +26,47 @@ def make_docs(chip):
     return chip
 
 
+def assemble_options(chip, tool):
+
+    step = chip.get('arg', 'step')
+    index = chip.get('arg', 'index')
+    # task = chip._get_task(step, index)
+
+    options = []
+
+    topmodule = chip.top()
+    blif = f"inputs/{topmodule}.blif"
+
+    for arch in chip.get('fpga', 'arch'):
+        options.append(arch)
+
+    options.append(blif)
+
+    if 'sdc' in chip.getkeys('input'):
```

Comment:
input/constraints/sdc or xdc? and why put it in FPGA vs. constraint?

Reply:
This is inherited from previous developments.  All that needs to happen here is that when an SDC file is provided, the command line option to pass the SDC file is added to the VPR command line, and when no SDC is provided, the option is omitted and we run without one.